### ISSUE TITLE
VxMark: Prompt ATI user that they have made max number of selections and can navigate to next contest

### ISSUE THREAD

**Issue by mattroe** [votingworks/vxsuite#5315]
https://votingworks.slack.com/archives/C01RCA4H3RV/p1723634944591839

Ben (summarised from NH a11y testing):
> Helpful if you have made the allowed number of selections in a contest that a window pops up and asks, "Would you like to move to the next contest or continue reviewing contest X?" That way you don't get stuck in a list of names or accidentally trigger the overvote warning.

Roe:
> tracking — to explicitly prompt ATI user that they have completed and can navigate (beyond just saying zero selections left) a la the color change we have for visual users

**Comment by kofi-q** [comment]
Any preferences on what the wording/interaction should be here?
Sounds like:
- For multi-seat contests, keep reading out `number of remaining votes`
- For both single-seat contests and final votes on multi-seat, read out `you're all done for this contest - press the right arrow button to advance to the next contest`

Or are we thinking a modal type of interaction with explicit options to advance or continue reviewing?

**Comment by mattroe** [comment]
@kofi-q this can be audio-tactile-only and doesn't need to be exposed in the visual-touch experience.

I suggest:
- keep reading out the number of remaining as you suggested
- when max number have been entered, reading out "You've completed your selections on this contest. Press the right arrow button to advance to the next contest. You may continue navigating in this contest to change your selections."

Open to alternative ideas if that sounds wordy too.

PS - when did you become a red panda?!

**Comment by kofi-q** [comment]
Thanks! Wording sounds good to me - I think once they've heard it all the way through once, they'll probably only listen for that first sentence before continuing.

And I've always been a red panda - just decided to make it officially public yesterday 😆  (mostly, I was having a hard time telling which default icon was mine on the issue boards lol)
